backend/src/common/helpers.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ *
+ * @param date Used when calling new Date() on the backend to query for availability events saved with a PST date string
```

Comment:
Can you add a comment here to avoid using this when SAVING dates?

Reply:
For sure.